app/src/main/java/com/example/concordia_campus_guide/Helper/CurrentLocationPermissionRequest.java

```diff
@@ -0,0 +1,56 @@
+package com.example.concordia_campus_guide.Helper;
+
+import android.Manifest;
+import android.content.Context;
+import android.content.pm.PackageManager;
+
+import com.example.concordia_campus_guide.ClassConstants;
+import com.example.concordia_campus_guide.Fragments.LocationFragment.LocationFragment;
+
+import androidx.core.app.ActivityCompat;
+import androidx.fragment.app.Fragment;
+
+import static androidx.core.content.ContextCompat.checkSelfPermission;
+
+public class CurrentLocationPermissionRequest extends Fragment {
+
+//    @Override
```

Comment:
why is this commented out?

Reply:
Because I amnot 100% if i will need this method or not. I will remove it if i see that i don't need at the end of the refactoring process .